rsearch/processor.go

```diff
@@ -0,0 +1,97 @@
+package rsearch
+
+import (
+	"log"
+)
+
+/*
+{"type":"ADDED","object":{"apiVersion":"romana.io/demo/v1","kind":"NetworkPolicy","metadata":{"name":"pol1","namespace":"default","selfLink":"/apis/romana.io/demo/v1/namespaces/default/networkpolicys/pol1","uid":"d7036130-e119-11e5-aab8-0213e1312dc5","resourceVersion":"119875","creationTimestamp":"2016-03-03T08:28:00Z","labels":{"owner":"t1"}},"spec":{"allowIncoming":{"from":[{"pods":{"tier":"frontend"}}],"toPorts":[{"port":80,"protocol":"TCP"}]},"podSelector":{"tier":"backend"}}}}
```

Comment:
If this serves as documentation then:
* Let's improve the formatting so that it can actually be read.
* Let's add some comments around it, so the reader knows what this is about.